src/models/msg.js

```diff
@@ -13,6 +13,7 @@ class Msg {
 			text: "",
 			type: Msg.Type.MESSAGE,
 			self: false,
+			users: [], // {start, end, nick}
```

Comment:
Hm, I don't think we need to set a default? `parse` should be able to deal with it just fine.

Also, it might be beneficial to rename this to `nicks` to be more specific.

Reply:
I'm getting the following, which might be related to your comment:

```
lounge/src/models/chan.js:124
	return this.users.get(nick.toLowerCase());
	                          ^

TypeError: Cannot read property 'toLowerCase' of undefined
    at Chan.findUser (/Users/jeremie/Perso/lounge/src/models/chan.js:124:28)
    at Chan.getUser (/Users/jeremie/Perso/lounge/src/models/chan.js:128:14)
    at IrcClient.<anonymous> (/Users/jeremie/Perso/lounge/src/plugins/irc-events/topic.js:18:15)
    at IrcClient.emit (/Users/jeremie/Perso/lounge/node_modules/eventemitter3/index.js:129:35)
    at /Users/jeremie/Perso/lounge/node_modules/irc-framework/src/client.js:185:20
    at next (/Users/jeremie/Perso/lounge/node_modules/middleware-handler/index.js:42:18)
    at MiddlewareHandler.handle (/Users/jeremie/Perso/lounge/node_modules/middleware-handler/index.js:64:3)
    at IrcCommandHandler.<anonymous> (/Users/jeremie/Perso/lounge/node_modules/irc-framework/src/client.js:179:34)
    at IrcCommandHandler.emit (/Users/jeremie/Perso/lounge/node_modules/eventemitter3/index.js:130:35)
    at IrcCommandHandler.emit (/Users/jeremie/Perso/lounge/node_modules/irc-framework/src/commands/handler.js:95:23)
```

Reply:
Doesn't appear related to the PR.

Reply:
It's not happening on `master`, so I'm guessing something gets passed wrong somewhere up the stack.

Reply:
yeah, this works fine without the default